src/main/features/claim/views/check-and-send.njk

```diff
@@ -38,36 +38,50 @@
         )
     }}
 
-    {{ row('Email:', draftClaim.defendant.email.address, paths.defendantEmailPage.uri) }}
+    {{ row('Email:', draftClaim.defendant.email.address, paths.defendantEmailPage.uri, bold = true) }}
     {{ tableEnd() }}
 
     {{ tableStart('claimAmount', 'Claim amount') }}
-    {{ row('Claim amount:', claimAmountTotal.claimAmount | numeral, paths.amountPage.uri )}}
+    {{ row('Claim amount:', claimAmountTotal.claimAmount | numeral, paths.amountPage.uri, bold = true) }}
 
-    {% if (interestClaimed === true) %}
-      {% if (draftClaim.interest.rate) %}
-        {{ row('Interest rate:', draftClaim.interest.rate | numeral('0,0.00') + '\% \(' + draftClaim.interest.type + '\)' , paths.interestPage.uri) }}
+    {% if (draftClaim.interest.option === InterestOption.YES) %}
+      {{ row('Claim Interest', 'Yes' , paths.interestPage.uri, bold = true) }}
+      {% if (draftClaim.interestType.option === InterestTypeOption.SAME_RATE) %}
+        {{ row('How do you want to claim interest?', 'Same rate for the whole period' , paths.interestRatePage.uri, bottomBorder = false, bold = true) }}
       {% endif %}
-      {% if payAtSubmission === false %}
-        {{ row('Date interest is claimed from:', draftClaim.interestDate.date.asString() | date, paths.interestDatePage.uri) }}
+      {% if (draftClaim.interestRate.type) %}
+        {{ row('What rate of interest do you want to claim?', draftClaim.interestRate.rate | numeral('0,0.00') + '\%', bottomBorder = false, bold = true) }}
```

Comment:
is `numeral('0,0.00')` not our default format?
i.e. what happens if you do just: `numeral`

Reply:
it adds a £ sign if I remove the passed in formatting